tfhe/src/integer/server_key/radix_parallel/scalar_comparison.rs

```diff
@@ -260,7 +260,7 @@ impl ServerKey {
             return vec![];
         }
 
-        debug_assert!(lhs.iter().all(|block| block.carry_is_empty()));
+        debug_assert!(lhs.iter().all(Ciphertext::carry_is_empty));
```

Comment:
yeah really not a fan of this one, does not make things more readable, do they say if performance can suffer from the old pattern ?